targets/linux/deb/distro/install.go

```diff
@@ -164,7 +170,7 @@ func (d *Config) InstallBuildDeps(sOpt dalec.SourceOpts, spec *dalec.Spec, targe
 			return in.Run(
 				dalec.WithConstraints(opts...),
 				customRepos,
-				InstallLocalPkg(pkg, opts...),
+				InstallLocalPkg(pkg, false, opts...),
```

Comment:
When are we expecting `upgrade=true` to be used?

Reply:
Creating the container.

Reply:
Ok, that makes sense!

Reply:
Oh I see, I'd missed the diff in `container.go`